src/main/groovy/com/readytalk/gradle/js/tasks/AsarPack.groovy

```diff
@@ -0,0 +1,23 @@
+package com.readytalk.gradle.js.tasks
+
+import com.moowork.gradle.node.task.NodeTask
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.OutputFile
+
+class AsarPack extends NodeTask {
+  @InputDirectory
+  def File inputDir
+
+  @OutputFile
+  def File outputFile
+
+  @Override
+  void exec() {
+    outputFile.parentFile.mkdirs();
+
+    setScript(project.file('node_modules/asar/bin/asar'))
```

Comment:
This does currently assume you have the asar package set up in your package.json file. I am open to suggestions on how best to manage that. I can have it install a certain version of the asar task, add it to the package.json of the project, etc. There are many options here, so let me know what you guys think would be best.